en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+2. Flash the firmware for your current release or master (PX4 `main` branch build).
+3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe.
+   At time of writing, this must be the [Generic VTOL Tailsitter](../airframes/airframe_reference.md#vtol-tailsitter).
+
+
+### Flight / Transition Mode Switch
+
+You should assign a switch on your RC controller for changing between the multicopter and fixed wing modes.
+
+:::note
+While PX4 allows flight without an RC controller, you must have one when tuning/configuring a new airframe.
+:::
+
+This is done in [Flight Mode](../config/flight_mode.md) configuration, where you [assign flight modes and other functions](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to switches on your RC controller.
+The switch can also be assigned using the parameter [RC_MAP_TRANS_SW](../advanced_config/parameter_reference.md#RC_MAP_TRANS_SW).
+
+The switch in the off-position means that you are flying in multicopter mode.
+
+## Duo Tailsitter MC Mode Tuning Guide
+
+MC-mode tuning is dificult because some axes depend on control surfaces, while others depend on the motors.
+
+:::warning
+Always disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) when tuning a vehicle.
+:::
+
+As a sanity check, we start off by setting the following gain parameters:
+
+- Roll rate control:
+  - [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) = [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) = [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D) = `0`, 
+  - [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K) = `1`
+- Pitch rate control
+  - [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = `0`
+  - [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = `1`
+- Yaw rate control
+  - [MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P) = [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I) = [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D) = `0`
+  - [MC_YAWRATE_K](../advanced_config/parameter_reference.md#MC_YAWRATE_K) = `1`
+- Angle control
+  - [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) = [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) = [MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P) = `0`
+
+The expected behaviour is that when you arm in [Stabilized mode](../flight_modes/manual_stabilized_mc.md), both the motors should spin, control surfaces should be centred, and no effect of manual RC input using the transmistter is observed.
+
+:::note IF you can do so safely!
+Gently pick up the vehicle by hand and try tilting it in different axes.
```

Comment:
Do we need a top level comment about how this process is manual and you need to be careful of props?

Reply:
Uhh yes. I think that is needed here. I was using glasses, a helmet and very thick gloves while doing this. Will probably mention that and also attach a video.

Reply:
Great. I've proposed a warning here: https://github.com/PX4/PX4-user_guide/pull/2593/files#r1252540336

Reply:
I've committed my suggestion. Might need stronger/weaker warning in future.